src/ode_solver/explicit_ode_solver.cpp

```diff
@@ -88,17 +93,60 @@ void ExplicitODESolver<dim,real,MeshType>::step_in_time (real dt, const bool pse
         }
 
         this->dg->solution = this->rk_stage[3];
+        this->current_time += dt;
         if ((ode_param.ode_output) == Parameters::OutputEnum::verbose) {
             this->pcout<< "done." << std::endl;
         }
     }
+    else if (rk_order == 4) {
```

Comment:
@cpethrick refactored this code so make the appropriate updates to this file considering her additions. She recently added RK4 to the code using butcher tables.

Reply:
fixed